### ISSUE TITLE
Build fails on 4.15

### ISSUE THREAD

**Issue by jeevcat** [Koderz/UE4RuntimeMeshComponent#30]
I'm not sure if this is related to me using UE 4.15, but when I try to compile I get the following error:

```
engine\source\runtime\coreuobject\public\UObject/Class.h(662): error C2280: 'FRuntimeMeshComponentPrePhysicsTickFunction &FRuntimeMeshComponentPrePhysicsTickFunction::operator =(const FRuntimeMeshComponentPrePhysicsTickFunction &)': attempting to reference a deleted function
Plugins\UE4RuntimeMeshComponent\Source\RuntimeMeshComponent\Public\RuntimeMeshComponent.h(69): note: compiler has generated 'FRuntimeMeshComponentPrePhysicsTickFunction::operator =' here
engine\source\runtime\coreuobject\public\UObject/Class.h(997): note: see reference to function template instantiation 'bool CopyOrNot<CPPSTRUCT>(CPPSTRUCT *,const CPPSTRUCT *,int32)' being compiled
        with
        [
            CPPSTRUCT=FRuntimeMeshComponentPrePhysicsTickFunction
        ]
engine\source\runtime\coreuobject\public\UObject/Class.h(996): note: while compiling class template member function 'bool UScriptStruct::TCppStructOps<FRuntimeMeshComponentPrePhysicsTickFunction>::Copy(void *,const void *,int32)'
Plugins\UE4RuntimeMeshComponent\Intermediate\Build\Win64\UE4Editor\Inc\RuntimeMeshComponent\RuntimeMeshComponent.generated.cpp(115): note: see reference to class template instantiation 'UScriptStruct::TCppStructOps<FRuntimeMeshComponentPrePhysicsTickFunction>' being compiled
```

Thanks for the great work. Would love to try this out on 4.15.

**Comment by Zinglish** [comment]
See this [Pull Request](https://github.com/Koderz/UE4RuntimeMeshComponent/pull/28) from @11011001. I believe he fixed some issues with building on UE 4.15.

**Comment by jeevcat** [comment]
Thanks. The latest commit (7c8f246e19b2cc78a51afdcd3b5ee3507fc7eb9f) fixed this for me.
